### PR TITLE
Removes some unnecessary species mentions from kapuparts

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
#65523 made some changes to bodypart names that messed with the grammar of a few messages around the codebase by inserting the species of a bodypart where it's really not needed. For example:

For most general combat/health cases, we don't need to worry about the species of a bodypart. So, this changes combat and wound messages back to simply displaying the relevant bodypart zone being attacked/affected. See below:

[](https://i.imgur.com/nrGXvyK.png)

Let me know if I missed any!

## Why It's Good For The Game
Better grammar

## Changelog

:cl: Ryll/Shaps
spellcheck: Wound and combat messages will no longer mention the species of an attacked bodypart when not appropriate
/:cl:

### CODE REVIEW

File: code/modules/surgery/bodyparts/robot_bodyparts.dm [hunk 1]
```diff
@@ -346,7 +346,7 @@
 	icon = 'icons/mob/augmentation/surplus_augments.dmi'
 	brute_reduction = 0
 	burn_reduction = 0
-	max_damage = 20
+	max_damage = 2026
```
Comment (cacogen) [inline code/modules/surgery/bodyparts/robot_bodyparts.dm:349]: based

Comment (Ryll-Ryll) [reply code/modules/surgery/bodyparts/robot_bodyparts.dm:349]: Oh my god thank you for catching that

Comment (Mothblocks) [reply code/modules/surgery/bodyparts/robot_bodyparts.dm:349]: um erm um

File: code/modules/mob/living/carbon/human/species_types/android.dm [hunk 2]
```diff
@@ -58,7 +58,7 @@
 		limb.skin_tone ||= skintone
 		limb.limb_id = SPECIES_HUMAN
 		limb.should_draw_greyscale = TRUE
-		limb.name = "human [parse_zone(limb.body_zone)]"
+		limb.name = "human [limb.plaintext_zone]"
```
Comment (Watermelon914) [inline code/modules/mob/living/carbon/human/species_types/android.dm:61]: same thing here

Comment (Kapu1178) [reply code/modules/mob/living/carbon/human/species_types/android.dm:61]: this is intentional, androids are supposed to have faux human limbs.

File: code/modules/mob/living/carbon/human/species_types/zombies.dm [hunk 3]
```diff
@@ -172,7 +172,7 @@
 			limb.skin_tone ||= skintone
 			limb.limb_id = SPECIES_HUMAN
 			limb.should_draw_greyscale = TRUE
-			limb.name = "human [parse_zone(limb.body_zone)]"
+			limb.name = "human [limb.plaintext_zone]"
```
[the remark this answers is not in the document]
Comment (Kapu1178) [reply code/modules/mob/living/carbon/human/species_types/zombies.dm:175]: half of this loop needs to go, funnily enough.

**Pull request merged**